sympy_bot/submodules.txt

```diff
@@ -17,6 +17,7 @@ core
 crypto
 diffgeom
 discrete
+equation
```

Comment:
Most of the other submodule names are plural. Shouldn't this be `equations` instead?